scripts/variantstore/wdl/extract/build_build_base_docker.sh

```diff
@@ -15,11 +15,35 @@ fi
 set -o errexit -o nounset -o pipefail -o xtrace
 
 BASE_REPO="broad-dsde-methods/variantstore"
-REPO_WITH_TAG="${BASE_REPO}:${1}"
+TAG="${1}"
+REPO_WITH_TAG="${BASE_REPO}:${TAG}"
 GCR_TAG="us.gcr.io/${REPO_WITH_TAG}"
 
+set +o errexit
+grep "${TAG}" <(docker images | grep "^${BASE_REPO}" | awk '{print $2}') > /dev/null
+RC="$?"
+set -o errexit
+
+if [[ $RC -eq 0 ]]
+then
+  echo "Error: Tag '${TAG}' already exists for image '${BASE_REPO}', refusing to tag again."
+  exit 1
```

Comment:
smart--but also wont this get really annoying if we want to overwrite a docker image that we just made in error?

Reply:
ok ignore me, I just saw the next section about deleting it!